### PR TITLE
Secure taker's routes with basic auth

### PR DESCRIPTION
**Pull request by thomaseizinger**
Tested locally and could open and collab close a CFD.

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]
nice one! smaller diff will hopefully make it easier to backport :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   The new payout transactions spend from the same lock transaction, so the rollover happens off-chain.
   In case a maker rejects a rollover request from a taker the old oracle price event and payout transactions stay in place.
 
+### Changed
```
Comment (da-kami) [inline CHANGELOG.md:23]: Should we mention that the taker's routes are secured by basicauth now?

**Review by da-kami — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (thomaseizinger) [reply CHANGELOG.md:23]: Yes!

**Pull request merged**